app/models/user.rb

```diff
@@ -0,0 +1,26 @@
+class User < ApplicationRecord
+  before_save :downcase_email
+
+  validates :email, presence: true, uniqueness: {case_sensitive: false},
```

Comment:
case_sensitive là gì em

Reply:
case_sensitive là một option của uniqueness để kiểm tra theo chữ thường và chữ in hoa. Cụ thể, dat@gmail.com sẽ khác với Dat@gmail.com nếu để case_sensitive: true